src/pages/App.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react';
+import logo from './logo.svg';
+import '../styles/App.scss';
+import { FrontEndMentorHeader } from '../components/FrontEndMentor';
+import { Filter } from '../components/Filter';
+import { Roadmap } from '../components/Roadmap';
+import { ToolBar } from '../components/ToolBar';
+import { RequestCards } from '../components/RequestCards';
+import { data } from 'data';
+import { ProductRequest } from 'types';
+
+const App = () => {
+  const [allData, setAllData] = useState(data);
+  const [showData, setShowData] = useState<ProductRequest[]>(
+    allData.productRequests,
+  );
+
+  const { productRequests } = allData;
+
+  return (
+    <div className="App">
+      <div className="App-col one">
```

Comment:
There's another HTML tag we could use here instead of a `div`. Any thoughts?

Reply:
Sections! Which I implemented

Reply:
I would maybe do navigation, but it's not really navigating to another page. it's just filter options. maybe an aside?

Reply:
Bingo - `<aside>` feels the most natural

Reply:
ok changed!